tortoise/tortoise.go

```diff
@@ -486,9 +483,15 @@ func (t *turtle) verifyLayers() error {
 				log.Stringer("hare", block.hare),
 				log.Stringer("persisted", block.persisted),
 			)
-			err := t.updater.UpdateBlockValidity(block.id, target, block.validity == support)
-			if err != nil {
-				return fmt.Errorf("saving validity for %s: %w", block.id, err)
+			valid := block.validity == support
+			if valid {
+				if err := blocks.SetValid(t.cdb, block.id); err != nil {
+					return fmt.Errorf("setting %s valid: %w", block.id, err)
+				}
+			} else {
+				if err := blocks.SetInvalid(t.cdb, block.id); err != nil {
+					return fmt.Errorf("setting block %s invalid: %w", block.id, err)
```

Comment:
this wrapping is completely unnecessary, it doesn't add any new information

but i think this change will have to be reverted